src/components/materialization/OnIncompatibleSchemaChange/Form.tsx

```diff
@@ -21,6 +26,8 @@ export default function SpecificationIncompatibleSchemaChangeForm() {
         async (option?: AutoCompleteOption | null) => {
             setFormState({ status: FormStatus.UPDATING, error: null });
 
+            setIncompatibleSchemaChange(option?.val);
```

Comment:
Not sure what the best UX here is. If the network call fails to update the draft then this new value is still in the store and rendered to the user as if the value changed.

I think we might need to make a quick `ref` and just set the value back if it fails.

Reply:
This is a strange defect. The stored copy of the drafted specification has an `onIncompatibleSchemaChange` value that is equivalent to the selected value despite the fact that `mutate` was not called by the hook specified in `updateServer`.

Reply:
This issue should be resolved but I am going to leave this thread open in the event you have additional comments.